README.md

```diff
@@ -26,19 +26,21 @@ Go to the [#skill-software-devops](https://helpfulengineering.slack.com/archives
+| Airtable Support | **[@psprings](https://helpfulengineering.slack.com/team/U010AK510F2)** in [#skill-software-devops](https://app.slack.com/client/TUTSYURT3/CV54M16QH)|
 | Auth0 | [@imonthercks](https://helpfulengineering.slack.com/team/U01063BPVHV) in [#skill-software-devops](https://app.slack.com/client/TUTSYURT3/CV54M16QH)|
-| Amazon Web Services (AWS) | [@skuenzli](https://helpfulengineering.slack.com/team/UV4C6N0M9), [@imonthercks](https://helpfulengineering.slack.com/team/U01063BPVHV) and [@Chris Mac](https://helpfulengineering.slack.com/team/U0101D9734M) in [#skill-software-devops](https://app.slack.com/client/TUTSYURT3/CV54M16QH)|
-| Azure | [@Troy Witthoeft](https://helpfulengineering.slack.com/team/UUXNLFC83) in [#skill-software-devops](https://app.slack.com/client/TUTSYURT3/CV54M16QH)|
+| Amazon Web Services (AWS) | [@skuenzli](https://helpfulengineering.slack.com/team/UV4C6N0M9) and [@imonthercks](https://helpfulengineering.slack.com/team/U01063BPVHV) in [#skill-software-devops](https://app.slack.com/client/TUTSYURT3/CV54M16QH)|
+| Azure | [@Praveen Karnati](https://helpfulengineering.slack.com/team/U01AR9RHLQ1), [@Priyanka](https://helpfulengineering.slack.com/team/U01ALPJ01S6) and [@Troy Witthoeft](https://helpfulengineering.slack.com/team/UUXNLFC83) in [#skill-software-devops](https://app.slack.com/client/TUTSYURT3/CV54M16QH)|
 | CRM / Odoo | [@Matt Melton](https://helpfulengineering.slack.com/team/U0109QJ8ALV) and [@Ray Carnes](https://helpfulengineering.slack.com/team/U010HQK88MU) in [#discussion-odoo-resources](https://app.slack.com/client/TUTSYURT3/G010L29KJ4E)|
-| Github Support | [@Frank Prins](https://helpfulengineering.slack.com/team/U010740NECE) and [@0x2b3bfa0](https://helpfulengineering.slack.com/team/U0103B34HGR) in [#skill-software-devops](https://app.slack.com/client/TUTSYURT3/CV54M16QH)|
+| Github Support | [@0x2b3bfa0](https://helpfulengineering.slack.com/team/U0103B34HGR) in [#skill-software-devops](https://app.slack.com/client/TUTSYURT3/CV54M16QH)|
 | Google Analytics |  |
-| Google Cloud Platform (GCP) | [@Rebecca Johns](https://app.slack.com/team/U0108S72KL4) in [#skill-software-devops](https://app.slack.com/client/TUTSYURT3/CV54M16QH)|
-| [Helpful main web site](https://helpfulengineering.org) | [@Charles He](https://helpfulengineering.slack.com/team/UUSFSSUTD) |
-| 1Password (secure password storage)| Read the [credential sharing guide](/documentation/guidance/credential-sharing.md) or message [@fparks](https://helpfulengineering.slack.com/team/UV23KD9MY) in [#skill-infosec](https://app.slack.com/client/TUTSYURT3/CV4TYGC1Z)|
+| Google Cloud Platform (GCP) | [@0x2b3bfa0](https://helpfulengineering.slack.com/team/U0103B34HGR) and [@dmangot](https://helpfulengineering.slack.com/team/UV9R7QTU3) in [#skill-software-devops](https://app.slack.com/client/TUTSYURT3/CV54M16QH)|
```

Comment:
```suggestion
| Google Cloud Platform (GCP) and G Suite | [@0x2b3bfa0](https://helpfulengineering.slack.com/team/U0103B34HGR) and [@dmangot](https://helpfulengineering.slack.com/team/UV9R7QTU3) in [#skill-software-devops](https://app.slack.com/client/TUTSYURT3/CV54M16QH)|
```
@dmangot, I've mixed both G Suite and GCP. Do you have any objection about being on that list until we get more volunteers?